src/app/ReadClient.h

```diff
@@ -126,6 +127,16 @@ class ReadClient : public Messaging::ExchangeDelegate
          */
         virtual void OnSubscriptionEstablished(SubscriptionId aSubscriptionId) {}
 
+        /**
+         * OnResubscriptionAttempt will be called when a re-subscription has been scheduled as a result of the termination of an
+         * in-progress or previously active subscription. This object MUST continue to exist after this call is completed. The
+         * application shall wait until it receives an OnDone call to destroy the object.
+         *
+         * @param[in] aTerminationCause The cause of failure of the subscription that just terminated.
+         * @param[in] aNextResubscribeIntervalMsec tell How long we will wait before trying to auto-resubscribe.
```

Comment:
```suggestion
         * @param[in] aNextResubscribeIntervalMsec How long we will wait before trying to auto-resubscribe.
```
Again, this was marked resolved but not done.

Please don't do that.  Manually double-checking every single thing to make sure it was actually fixed is a huge time sink.  :(

Reply:
sorry about that. I thought i have locally made this change, but somehow forgot to git add them. would manually do scrutiny , thanks